vpr/src/pack/post_routing_pb_pin_fixup.cpp

```diff
@@ -153,49 +153,55 @@ static void update_cluster_pin_with_post_routing_results(const Netlist<>& net_li
         ParentNetId routing_net_id = ParentNetId::INVALID();
         std::vector<RRNodeId> visited_rr_nodes;
         short valid_routing_net_cnt = 0;
-        for (const e_side& pin_side : pin_sides) {
-            /* Find the net mapped to this pin in routing results */
-            RRNodeId rr_node = node_lookup.find_node(grid_coord.x(), grid_coord.y(), rr_node_type, physical_pin, pin_side);
-
-            /* Bypass invalid nodes, after that we must have a valid rr_node id */
-            if (!rr_node) {
-                continue;
-            }
-            VTR_ASSERT((size_t)rr_node < device_ctx.rr_graph.num_nodes());
-
-            /* If the node has been visited on the other side, we just skip it */
-            if (visited_rr_nodes.end() != std::find(visited_rr_nodes.begin(), visited_rr_nodes.end(), RRNodeId(rr_node))) {
-                continue;
-            }
-
-            /* Get the cluster net id which has been mapped to this net
-             * In general, there is only one valid rr_node among all the sides.
-             * However, we have an exception in the Stratix-IV arch modeling,
-             * where a pb_pin may exist in two different sides but
-             * router will only map to 1 rr_node 
-             * Therefore, it is better to compare the routing nets
-             * for all the sides and pick
-             * - The unique valid net id (others should be all invalid)
-             *   assume that this pin is used by router
-             * - A invalid net id (others should be all invalid as well)
-             *   assume that this pin is not used by router
-             */
-            if (rr_node_nets[rr_node]) {
-                if (routing_net_id) {
-                    if (routing_net_id != rr_node_nets[rr_node]) {
-                        VTR_LOG_ERROR("Pin '%s' is mapped to two nets: '%s' and '%s'\n",
-                                      pb_graph_pin->to_string().c_str(),
-                                      net_list.net_name(routing_net_id).c_str(),
-                                      net_list.net_name(rr_node_nets[rr_node]).c_str());
-                    }
-                    VTR_ASSERT(routing_net_id == rr_node_nets[rr_node]);
-                }
-                routing_net_id = rr_node_nets[rr_node];
-                valid_routing_net_cnt++;
-                visited_rr_nodes.push_back(rr_node);
+        int addToY = 0, addToX = 0;
+        addToY = physical_tile->height;
+        addToX = physical_tile->width;
```

Comment:
Better coding practice to avoid the dummy initialization to 0, and instead just directly say
int addToY = physical_tile->height;

(Shorter and more idiomatic).

Reply:
We apply your comments.